### DISCUSSION TITLE
Want to add multiple instances of wavesurfer js to my landing page

### DISCUSSION THREAD

**Discussion by liamf1995** [katspaugh/wavesurfer.js#2783] [Q&A]
I am trying to insert several instances of wavesurfer js onto my GemPages landing page, I can insert one instance just fine, the audio plays, buttons work perfectly

when I try to copy and past the code into a new liquid block, the "play/pause" buttons show up in the correct place, but, the wav form does not, it shows  up directly below the 1st instance of wavesurver js

and the audio does not play for either of them now,

please help!?

**Comment by katspaugh** [comment]
Hard to tell without seeing your code.

**Comment by samplethisdotnet** [comment]
Hey! Could you show me how you inserted the custom liquid block on gempages? I can't for the life of me get wavesurfer to show up on my gempages.
